pom.xml

```diff
@@ -54,6 +54,16 @@
             <artifactId>logstash-logback-encoder</artifactId>
             <version>6.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.sql2o</groupId>
+            <artifactId>sql2o</artifactId>
+            <version>1.5.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.xerial</groupId>
+            <artifactId>sqlite-jdbc</artifactId>
+            <version>3.8.11.2</version>

```

Comment:
maybe is possible to use a new version for this one too.

Reply:
I checked the version on https://mvnrepository.com/ , changing it to 3.32.3.2